src/bridge/cockpitmemorysamples.c

```diff
@@ -66,10 +67,14 @@ cockpit_memory_samples (CockpitSamples *samples)
         g_warn_if_fail (sscanf (line + sizeof ("Buffers:") - 1, "%" G_GUINT64_FORMAT, &buffers_kb) == 1);
       else if (g_str_has_prefix (line, "Cached:"))
         g_warn_if_fail (sscanf (line + sizeof ("Cached:") - 1, "%" G_GUINT64_FORMAT, &cached_kb) == 1);
+      else if (g_str_has_prefix (line, "MemAvailable:"))
+        g_warn_if_fail (sscanf (line + sizeof ("MemAvailable:") - 1, "%" G_GUINT64_FORMAT, &available_kb) == 1);
+
     }
 
   cockpit_samples_sample (samples, "memory.free", NULL, free_kb * 1024);
   cockpit_samples_sample (samples, "memory.used", NULL, (total_kb - free_kb) * 1024);
+  cockpit_samples_sample (samples, "memory.unavailable", NULL, (total_kb - available_kb) * 1024);
```

Comment:
I think it would be fine just to fix the way we compute `memory.used`.  No need to introduce a new metric, IMO.